django_project/frontend/src/containers/MainPage/Metrics/AreaAvailableLineChart.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useEffect, useState } from "react";
+import { Box, Typography } from "@mui/material";
+import { Line } from "react-chartjs-2";
+import { CategoryScale } from "chart.js";
+import Chart from "chart.js/auto";
+import Loading from "../../../components/Loading";
+import "./index.scss";
+
+Chart.register(CategoryScale);
+
+const AreaAvailableLineChart = (props: any) => {
+    const { loading, areaData } = props
+    const AreaDataValue = {
+        labels: areaData.map((item: any) => item?.annualpopulation__year),
+        datasets: [
+            {
+                label: "Area available to species",
+                data: areaData.map((item: any) => item?.area_available),
+                borderColor: "#F9A95D",
```

Comment:
Could we use color variables for these colors

Reply:
@dimasciput  do you mean like this ?
 const orange = "#F9A95D"
 borderColor: orange

Reply:
@ketankartoza use the color from here :
https://github.com/kartoza/sawps/blob/feature-area-available-line-chart/django_project/frontend/src/assets/styles/index.scss#L17
If it doesn't exist, then please add new one.

Reply:
@dimasciput I attempted to use a color variable, but it does not seem to be functioning properly when working with a canvas.